presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -732,3 +732,50 @@ The following query will fail with the error ``Column 'name' is ambiguous``::
     FROM nation
     CROSS JOIN region;
 
+
+Subqueries
+----------
+
+A subquery is an expression which is composed of a query. The subquery
+is correlated when it refers to columns outside of the subquery.
+Logically, the subquery will be evaluated for each row in the surrounding
+query. The referenced columns will thus be constant during any single
+evaluation of the subquery.
+
+.. note::
+
+  Support for correlated subqueries is limited. Not every standard form is
+  supported
```

Comment:
No need to wrap here for one word (there's no hard guideline -- just do what makes it most readable).

Also end the sentence in a period.